lib/modules/modules.go

```diff
@@ -73,6 +75,11 @@ func (p *defaultModules) EmptyRolesHandler() error {
 	return nil
 }
 
+// DefaultKubeUsers returns default kuberentes users for a new admin role
+func (p *defaultModules) DefaultKubeUsers() []string {
+	return []string{teleport.TraitInternalKubeUsersVariable}
+}
+
 // DefaultKubeGroups returns default kuberentes groups for a new admin role
```

Comment:
`kubernetes` (might as well fix this too)

Reply:
done